### DISCUSSION TITLE
父流水线可以通过变量的形式定义子流水线中stage和job的名称

### DISCUSSION THREAD

**Discussion by itliushuai** [Tencent/bk-ci#6525] [Ideas]
父流水线定义变量，并将变量透传给子流水线，子流水线根据透传的变量修改stage和job的名称

**Comment by zanyzhao** [comment]
1. 在子流水线里声明所需的变量，通过父流水线传进来不就行了？
2. 简单说下你通过变量改stage、job名称的目的
